static/app/utils/replays/replayReader.tsx

```diff
@@ -194,6 +197,18 @@ export default class ReplayReader {
 
   getRRWebFrames = () => this._sortedRRWebEvents;
 
+  getRRWebMutations = () =>
+    this._sortedRRWebEvents.filter(
+      event =>
+        [
+          EventType.DomContentLoaded,
+          EventType.Load,
+          EventType.FullSnapshot,
+          EventType.IncrementalSnapshot,
+        ].includes(event.type) &&
+        ('adds' || 'removes') in (event as incrementalSnapshotEvent).data // filter only for mutation events
```

Comment:
i'm not sure this is working the way it looks like it does.

`('adds' || 'removes')` should resolve to `true`
and then `true in event.data` is weird, probably allowing everything through because `in` is super old and coerces things/is not typesafe.

Reply:
woo resolved! and keeping the sample rate as an additional barrier against super long/high activity replays